### DISCUSSION TITLE
Temperature wave when a door is activated

### DISCUSSION THREAD

**Discussion by y00ucef** [firemodels/fds#13459] [Q&A] [answered]
Hello, 
![temperature_wave_begining](https://github.com/user-attachments/assets/99959abf-4be2-40ad-a704-6f2a36f0e655)
![Temperature_wave_activation](https://github.com/user-attachments/assets/08f66fdf-e591-45de-8109-ccbba20b8504)

I've simulated a fire on a hospital floor. At the beginning of the simulation, I observe hot spots that get hotter. In this simulation I also activate a door which allows to separate the damaged part from the hospital. This door is activated when a smoke detector is activated. When the door is activated, I observe a sort of temperature wave (image attached) in a non-disaster area. I don't know what causes this wave, and whether it makes my simulation false.

 Thank you in advance for your help.

**Comment by drjfloyd** [comment]
Without knowing the details of your simulation (i.e., the input file) can't really opine on what might be causing this.  Make sure you are running the latest version of FDS.

**Reply by y00ucef** [reply]
i used pyrosim to parameterize the simulation. you'll find attached a document containing all the lines as well as a simplified version containing the parameters.  I used FDS 6.8
Thank you very much.
[complete_code.txt](https://github.com/user-attachments/files/17098139/complete_code.txt)
[simplified_code.txt](https://github.com/user-attachments/files/17098140/simplified_code.txt)

**Reply by drjfloyd** [reply]
"Make sure you are running the latest version of FDS" 

6.8.0 is not the latest version. Try with the current release and see if you still observe the issue.

**Comment by Er9y714** [comment]
I have not checked the inputs but make sure the compartment is not sealed when all doors are closed. There should be some opening or at least a leak. Otherwise the pressure might unrealistically decrease/increase depending on your setup which might result in local temperature changes. You might also observe pressure to verify this.

**Comment by y00ucef** [comment, marked as answer]
Thanks i’ll rerun the case with leaks in each room. Thanks in advanced
